bin/src/storage.rs

```diff
@@ -42,17 +42,19 @@ pub fn generate_filters(
     let stopwords = String::from_utf8(bytes.to_vec())?;
     let stopwords: HashSet<String> = stopwords.split_whitespace().map(String::from).collect();
 
-    let split_posts: HashMap<PostId, HashSet<String>> = posts
+    let split_posts: HashMap<PostId, Option<HashSet<String>>> = posts
         .into_iter()
         .map(|(post, content)| {
             debug!("Generating {:?}", post);
             (
                 post,
-                cleanup(strip_markdown(&content))
-                    .split_whitespace()
-                    .map(str::to_lowercase)
-                    .filter(|word| !stopwords.contains(word))
-                    .collect::<HashSet<String>>(),
+                content.map(|content| {
```

Comment:
Nice pattern! I didn't know that `Option.map()` would leave the `None` case untouched.

Reply:
Yeah, it's pretty nice to use `Option` on things like this. (Same for `Result`)